### PR TITLE
Stale bot activation

### PR DESCRIPTION
**Pull request by LukasHeinz**
| Q             | A
| ------------- | ---
|Description, reason for the PR| There are issues on GitHub that are stale - there is no activity and probably there wont be in near future. But those issues cause going through issues very difficult.
|New feature| No 
|BC breaks| No 
|Fixes issues| -
|Standards and tests pass| -
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by LukasHeinz** [comment]
Long story short - after 60 days of inactive issue there will be "Stale" label and after another 90 days this issue will be closed

**Comment by LukasHeinz** [comment]
I have given stalebot the authorization for our repositories - https://github.com/organizations/shopsys/settings/installations/270550

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 #### Fixed
 - [#260 - JS validation: dynamically added form inputs are now validated](https://github.com/shopsys/shopsys/pull/260)
 
+### [shopsys/shopsys]
+#### Added
+- [#379 Stale bot activation](https://github.com/shopsys/shopsys/pull/379)
+    - added configuration file for GitHub Stale bot 
+
```
Comment (boris-brtan) [inline CHANGELOG.md]: please remove blank line

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
```
Comment (boris-brtan) [inline CHANGELOG.md]: please remove blank line

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+# Comment to post when closing a stale Issue or Pull Request.
+# closeComment: >
+  This issue has been automatically closed because it has not had
+  recent activity for a long time. 
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 30
+
+# Limit to only `issues` or `pulls`
```
Comment (boris-brtan) [inline .github/stale.yml]: Please remove these commented and unused parts. from line 43 till the last line.
All the configuration can be found in probot-stale repo, link is on the first line of this file `https://github.com/probot/stale`

**Comment by boris-brtan** [comment]
Thanks for the fixes, it looks perfect.

**Comment by PetrHeinz** [comment]
Hi, I've noticed the PR and I have two notes:
- Please use 4 spaces in YAML for indentation to be consistent with the rest of the app (see #389)
- Consider moving our [templates for pull requests and issues](https://help.github.com/articles/about-issue-and-pull-request-templates/) into the newly created `.github` directory, as they are used only for Github integration. We originally put the to `docs` to not have excessive directories in the root, just because of one template, now it makes much more sense...

**Pull request merged**